src/CheckArea/style.css

```diff
@@ -44,6 +43,8 @@
     & .ola_check {
         display: block;
         padding: 0;
+        background: var(--white);
+        border-radius: var(--radius-small);
```

Comment:
If we move the `background` to `.ola_check-label`, we don't need the extra border-radius here.